### PR TITLE
Wire event watcher for LateConnections events

### PR DESCRIPTION
**Pull request by mfojtik**
This wire event watcher for `LateConnections` event that increase the `openshift_kube_apiserver_lateconnections_count` prometheus metric.

A follow up would be to add Prometheus Alert.

Blocked on https://github.com/openshift/origin/pull/25126
/hold

### CODE REVIEW

**Comment by mfojtik** [comment]
(we don't have to wait for origin, this will be no-op until that PR merge and this can get some testing cycles meanwhile ;-)

File: pkg/operator/starter.go [hunk 1]
```diff
@@ -104,6 +106,10 @@ func RunOperator(ctx context.Context, controllerContext *controllercmd.Controlle
 		controllerContext.EventRecorder,
 	)
 
+	eventWatcher := eventwatch.New().
+		WithEventHandler(operatorclient.TargetNamespace, "LateConnections", terminationobserver.ProcessLateConnectionEvents).
```
Comment (soltysh) [inline pkg/operator/starter.go:110]: @mfojtik who's producing this event, asking for a friend :wink:

Comment (mfojtik) [reply pkg/operator/starter.go:110]: https://github.com/openshift/origin/pull/25126

**Pull request merged**